src/kubernetes_api_objects/persistent_volume_claim.rs

```diff
@@ -313,4 +313,98 @@ impl Marshalable for PersistentVolumeClaimSpecView {
     proof fn marshal_preserves_integrity() {}
 }
 
+#[verifier(external_body)]
```

Comment:
I am not sure whether you want to place `StatefulSetPersistentVolumeClaimRetentionPolicy` in this file. Probably better to move it to stateful_set.rs (since it is only used by statefulset)